locales/pt_BR/pcsx2_Iconized.po

```diff
@@ -8,309 +8,305 @@ msgstr ""
 "Project-Id-Version: PCSX2 1.6.0\n"
 "Report-Msgid-Bugs-To: https://github.com/PCSX2/pcsx2/issues\n"
 "POT-Creation-Date: 2020-02-10 17:29+0000\n"
-"PO-Revision-Date: 2020-20-19 18:26-0300\n"
-"Last-Translator: Altieres Lima a Silva <altieres.lima@gmail.com>; Rafael Fontenelle <rffontenelle@gmail.com>\n"
-"Language-Team: \n"
+"PO-Revision-Date: 2021-03-30 17:36-0300\n"
+"Last-Translator: Altieres Lima a Silva <altieres.lima@gmail.com>; Rafael "
+"Fontenelle <rffontenelle@gmail.com>\n"
+"Language-Team: Felipe\n"
 "Language: pt_BR\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n > 1);\n"
-"X-Generator: Poedit 2.2.4\n"
+"X-Generator: Poedit 2.4.2\n"
 "X-Poedit-KeywordsList: pxE;pxEt\n"
-"X-Poedit-SourceCharset: utf-8\n"
+"X-Poedit-SourceCharset: UTF-8\n"
 
 #: common/src/Utilities/Exceptions.cpp:239
 msgid ""
 "There is not enough virtual memory available, or necessary virtual memory "
 "mappings have already been reserved by other processes, services, or DLLs."
 msgstr ""
+"Não há memória virtual suficiente disponível ou os mapeamentos da memória "
+"virtual necessários já foram reservados por outros processos, serviços ou DLLs."
 
 #: pcsx2/CDVD/CDVD.cpp:2058
 msgid ""
 "Failed to read/write NVM/MEC file. Check your BIOS setup/permission settings."
 msgstr ""
-"Falha ao ler ou escrever o arquivo NVM/MEC. Verifique suas configurações BIOS e "
-"de permissões."
+"Falhou em ler/gravar o arquivo NVM/MEC. Verifique suas configurações da BIOS/"
+"configurações das permissões."
 
 #: pcsx2/System.cpp:112
 msgid ""
-"This recompiler was unable to reserve contiguous memory required for "
-"internal caches.  This error can be caused by low virtual memory resources, "
-"such as a small or disabled swapfile, or by another program that is hogging "
-"a lot of memory."
+"This recompiler was unable to reserve contiguous memory required for internal "
+"caches.  This error can be caused by low virtual memory resources, such as a "
```

Comment:
One of them

Reply:
I see, i wonder how this is possible since with poedit i can only translate without changing the code.

Reply:
Hence why we are changing to this system because poedit can act odd at times, no matter what OS you have or locale.
Feature freeze is coming closer and closer everyday, i know translators have their patience been tested for quite sometime, but i feel after this pain it will feel much better. So please bear it with me.

Reply:
Ok, i'll wait, thanks for the explanations.